octoprint_smsnotifier/.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+max-line-length= 120
+
+ignore=
+ E127, E128 # bad indentation for continuation lines; would result in really deep indentation, which is the whole reason for doing continuation lines.
```

Comment:
This .flake8 is specific to your IDE? Maybe we should remove it?

Reply:
vim is my IDE; I have it to make sure I keep to your indentation style and such. So it's specific to this repo.